ignite/src/main/resources/interpreter-setting.json

```diff
@@ -0,0 +1,34 @@
+[
+  {
+    "group": "ignite",
+    "name": "ignite",
+    "className": "org.apache.zeppelin.ignite.IgniteInterpreter",
+    "properties": {
+      "ignite.addresses": {
+        "envName": null,
+        "propertyName": "ignite.addresses",
+        "defaultValue": "127.0.0.1:47500..47509,Coma separated list of addresses",
```

Comment:
`"Comma separated list of addresses"` should in description below instead of in the `defaultValue`